### DISCUSSION TITLE
Edit Filter compare between 2 fields

### DISCUSSION THREAD

**Discussion by minhhungit** [serenity-is/Serenity#5658] [Help]
hello it looks Edit Filter feature does not allow us compare between 2 fields right?
For example: City equal"Paris" and City equal Country 
Currently I have to enter value "Paris" for both field, like this:
```
fld.City equal "Paris"
and 
fld.Country equal "Paris"
```

It is better if we can do this:

```
fld.City equal "Paris"
and
fld.City equal fld.Country
```
 If we did not support it, any suggestion so I can implement it please

**Comment by erehtolleh** [comment]
Not exactly sure what you are referring but are you talking about cascading lookups?
https://serenity.is/docs/howto/how_to_setup_cascaded_editors

**Reply by minhhungit** [reply]
ah no I mean this feature

![image](https://user-images.githubusercontent.com/2279508/115917065-ded21f80-a49f-11eb-9206-033488ae450b.png)

**Reply by erehtolleh** [reply]
ah sorry you are too advance for me.  hopefully someone else knows..
